pages/domains/domains_benefits.md

```diff
@@ -13,7 +13,7 @@ eleventyNavigation:
 
 ## .Gov helps the public identify official, trusted information
 
-.Gov is a top-level domain like .com, .org, or .us. For a fee, anyone can register popular top-level domains like .com or .org. This can make it hard for the public to know if the people behind an online service are who they claim to be.
+Anyone can register top-level domains like .com, .org, or .us for a fee. This can make it hard for the public to know if the people behind an online service are who they claim to be.
```

Comment:
Blocking nit: Very few people can register _top-level domains_, though they can get a domain _in_ those TLDs for a fee.

Reply:
Ok, since we always spell out the acronym (top-level domains) that makes it trickier to use without the sentence sounding clunky and confusing. I made some revisions in the corresponding [Google doc](https://docs.google.com/document/d/1KYYOvy5adYu5vgmXBFXNXzD0m-Vt3GWp1rqs-9yOxxk/edit). Let me know what you think.

Note: we'll need to update the [Eligibility](https://beta.get.gov/domains/eligibility/) page if we're changing this language.

cc @michelle-rago